### PR TITLE
Fix docker container add anycast and connection type

### PR DESCRIPTION
**Pull request by lunascontributions**
- added support for the Connection Type database
- added support for new is_anycast flag
- changed docker command to clean up the container after use

### CODE REVIEW

File: bin/docker-run.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 set -eux
 
-docker run -e POSTGRES_PASSWORD=foo --rm -p 5432:5432 --volume "$PWD/share/dbs:/share/dbs" mmpg
+docker run -e POSTGRES_PASSWORD=foo --rm --shm-size=1g -p 5432:5432 --volume "$PWD/share/dbs:/share/dbs" mmpg
```
Comment (oalders) [inline bin/docker-run.sh]: Could you add a comment explaining why this change is necessary?

Comment (lunascontributions) [reply bin/docker-run.sh]: ah, i don't think that is the change that was necessary -- let me take a look

**Review by oalders — APPROVED** [review]
👍🏻
